Viper/Viper/CountryListInteractor/CountryListInteractorProtocol.swift

```diff
@@ -0,0 +1,11 @@
+// CountryListInteractorProtocol.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Протокол интерактора экрана списка стран
+protocol CountryListInteractorProtocol {
+    var networkService: NetworkServiceProtocol { get set }
+    var presenter: CountryListPresenterProtocol? { get set }
```

Comment:
```suggestion
    var networkService: NetworkServiceProtocol { get set }
    var presenter: CountryListPresenterProtocol? { get set }
```
тут тоже самое, проверь нужен ли сет, и в других протоколах так же.